### PR TITLE
use many threads for parallelizing calls that are mostly networking / IO

### PR DESCRIPTION
**Pull request by tkruse**
@wjwwood: what do you about this change?

### CODE REVIEW

File: src/wstool/multiproject_cmd.py [hunk 1]
```diff
@@ -345,7 +345,9 @@ def do_work(self):
                                  verbose=self.report.verbose)
             return {}
 
-    work = DistributedWork(len(preparation_reports), num_threads, silent=False)
+    work = DistributedWork(len(preparation_reports),
+                           num_threads=num_threads,
+                           silent=False)
```
Comment (wjwwood) [inline src/wstool/multiproject_cmd.py]: To be consistent with the other calls to `DistributedWork` it would be `capacity=len(...`.

**Comment by wjwwood** [comment]
I think it's probably a reasonable change. The previous default was 10 and now it's the number of items to be processed? I don't think that should be a problem, but it's hard to predict what will be a problem for people in practice.

I'm happy to merge it if you are.

**Comment by tkruse** [comment]
I added the argument name for consistency. The change is as you say. Maybe when this is released, we can point to that change and ask people to try it out and report problems. I don't expect any, other than if the implementation as bugs.

**Comment by wjwwood** [comment]
Ok, I'll merge it.

**Pull request merged**